src/cameras/capture/opencv_camera/frame_grabber.py

```diff
@@ -71,8 +71,8 @@ def _start_frame_loop(self, save_video=False):
         start = time.time()
         try:
             while self._is_capturing_frames:
-                success, image, timestamp = self._get_next_frame()
-                self._frame = FramePayload(success, image, timestamp)
+                success, image, timestamp, webcam_id = self._get_next_frame()
```

Comment:
lets change this to:

```suggestion
                frame = self._get_next_frame()
                self._frame = frame
```

and then use `frame` downstream in the method if necessary.

Reply:
works for me!

Reply:
my only concern is that this method makes this a little less explicit because you need to know (or figure out) that the `frame` object contains `success` `image` and `timestamp`.

... but I see why that's a good tradeoff, because it feel sloppy to create new variable if we don't know for certain we're going to use them

Reply:
if we've named a class `FramePayload`, it /should/ make sense that the instance of such a class may be named `frame` - thats the logic i'm employing atm.